LocalPackages/DataBrokerProtection/Sources/DataBrokerProtection/Scheduler/DataBrokerProtectionAgentManager.swift

```diff
@@ -34,7 +35,8 @@ public class DataBrokerProtectionAgentManagerProvider {
                                     accountManager: AccountManager) -> DataBrokerProtectionAgentManager {
         let pixelHandler = DataBrokerProtectionPixelsHandler()
 
-        let executionConfig = DataBrokerExecutionConfig()
+        let dbpSettings = DataBrokerProtectionSettings()
+        let executionConfig = DataBrokerExecutionConfig(mode: dbpSettings.storedRunType == .integrationTests ? .fastForIntegrationTests : .normal)
```

Comment:
Is there anyway we can add a test to ensure this is always set correctly in production environment.

Reply:
Possible but annoying, Would require mocking DBPSettings which it isn't designed to do. It's a good shout though, might be worth it?

Reply:
I think worth a time-boxed look to judge effort.

Reply:
Chatted with @THISISDINOSAUR on MM re: this and the changes needed to do this are outside the scope of this work.